agent/backend/pktvisor/pktvisor.go

```diff
@@ -17,7 +19,6 @@ import (
 
 	mqtt "github.com/eclipse/paho.mqtt.golang"
 	"github.com/go-cmd/cmd"
-	"github.com/go-co-op/gocron"
```

Comment:
Are we using it for anything else? If not, go dependency should be removed from go.mod

Reply:
was waiting the results of the report to send the clean up. here you go